### PR TITLE
docs: add korean README and add test method

### PR DESCRIPTION
**Pull request by SQLGate**
Add korean README and Add test method

### CODE REVIEW

File: README-zhCN.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Molecule 是一款受 **VSCode** 启发，使用 **React.js** 构建的 **Web ID
 -   基本兼容 Visual Studio Code 的 **ColorTheme**
 -   支持使用 React 组件自定义 **Workbench** UI 样式
 -   内置 Monaco Editor **Command Palette**、**Keybinding**等模块，并支持扩展
--   支持 **i18n**，内置简体中文、English 2 种语言
+-   支持 **i18n**，簡体字中国語、英制、朝鲜语 3 种语言
```
Comment (wewoor) [inline README-zhCN.md:32]:
```suggestion
-   支持 **i18n**，简体中文、English 、한국어 3 种语言
```

File: README.md [hunk 2]
```diff
@@ -17,19 +17,19 @@
...
 ## Features
 
--   Built-in the VSCode **Workbench** UI
--   Compatible with the VSCode **ColorTheme**
+-   Built-in the Visual Studio Code **Workbench** UI
+-   Compatible with the Visual Studio Code **ColorTheme**
 -   Customize the Workbench via **React Component** easily
 -   Built-in Monaco-Editor **Command Palette, Keybinding** features
--   Support the **i18n**, built-in zhCN, and English
+-   Support the **i18n**, built-in Simplified Chinese, Korean and English
```
Comment (wewoor) [inline README.md:32]: Sync these updates to files:

```bash
website/docs/introduction
website/docs/guides/extend-locales
website/i18n/zh-CN/current/introduction
website/i18n/zh-CN/current/guides/extend-locales
```

**Review by wewoor — CHANGES_REQUESTED** [review]
/docs/introduction

File: README.md (hunk 2 above)
Comment (SQLGate) [reply README.md:32]: I'm sorry, but do you plan to keep the documentation site in Chinese?
https://dtstack.github.io/molecule/en/docs

**Review by wewoor — APPROVED** [review]
+1

**Pull request merged**